docs/modules/ROOT/pages/getting-started/deploy/terraform.adoc

```diff
@@ -0,0 +1,14 @@
+
+= Deploy with Terraform
+
+Terraform supports a wide range of cloud providers.
```

Comment:
This part will need to be a snippet so that it can be included into the frontend